README.md

```diff
@@ -42,12 +44,59 @@ to query the vector store and review potential SAST errors.
 ### Evaluation
 - Applies metrics (from Ragas library) to assess the quality of model outputs.
 
-## Evaluation & Metrics 📊
+## 📊 Evaluation & Metrics
 The evaluations of the model responses are being done using the following metrics:
 - **Response Relevancy:**  
   Ensures that the generated answers are directly related to the query.  
   [Response Relevancy](https://docs.ragas.io/en/latest/concepts/metrics/available_metrics/answer_relevance/).
   
 
-## Installation & Setup 🔌
+## 🔌 Installation & Setup 
 Please refer to [how to run](./docs/setup.md) guideline.
+
+
+## 🚀 Running the Application in a Container (Locally)
+
+Follow these steps to build, push, and run your container image:
+
+1. **Build the Container Image:**
+
+Use the following command to build your image. Adjust the tag (`1.0.0-SNAPSHOT`) as needed based on project stage.
+
+  ```bash
+  podman build -t quay.io/ecosystem-appeng/sast-ai-workflow:1.0.0-SNAPSHOT -f deployment/containerfile .
+  ```
+
+2. **Log in to Quay:**
+
+Authenticate with your Quay credentials using:
+
+```bash
+podman login quay.io
+```
+
+3. **Push the Image:**
+
+Once authenticated, push your image to the Quay repository:
+
+```bash
+podman push quay.io/ecosystem-appeng/sast-ai-workflow:1.0.0-SNAPSHOT
```

Comment:
I think we don't need these steps. Because this is well known info and also only interested for the contributors. Anyone who simply wants to try this out will not be benefitted with this info.